packages/legend-manual-tests/src/__tests__/roundtrip-grammar/RoundtripGrammar.test.ts

```diff
@@ -88,6 +88,9 @@ const EXCLUSIONS: { [key: string]: ROUNTRIP_TEST_PHASES[] | typeof SKIP } = {
   'nested-embedded-relational-mapping.pure': SKIP,
   'relational-mapping-filter.pure': SKIP,
 
+  // TODO: remove this when https://github.com/finos/legend-engine/pull/519 is merged.
+  'ESServiceStore-pathOffset.pure': SKIP,
+
```

Comment:
Can you please change the file name to `ESService-path-offset.pure` to follow the convention.

Reply:
done